build/packages.targets

```diff
@@ -7,7 +7,6 @@
         <VSFrameworkVersion>16.6.30107.105</VSFrameworkVersion>
         <VSServicesVersion>16.153.0</VSServicesVersion>
         <VSThreadingVersion>16.6.13</VSThreadingVersion>
-        <!-- TODO - remove this when temporary patching is no longer necessary. See https://github.com/nuget/home/issues/8952 -->
         <PatchedSystemPackagesVersion>5.0.0-preview.3.20214.6</PatchedSystemPackagesVersion>
```

Comment:
You only removed the comment.
How about the property variable?

Reply:
Thanks for reviewing!
The property `PatchedSystemPackagesVersion` is still needed, as the System.Security.Cryptography.Pkcs/Cng are not in the .NET 5 Base Class Libraries (BCL). We need to keep the package referrence of System.Security.Cryptography.Pkcs/Cng.
So only the comment should be removed.

Reply:
Should we change the name to a property that communicates the intention better? 

Maybe something like CryptographyPackagesVersion or something similar.

Reply:
Thanks. Fixed.